website/content/docs/installation/index.mdx

```diff
@@ -50,16 +50,9 @@ VBoxManage: error: VirtualBox can't operate in VMX root mode. Please disable the
 VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
 ```
 
-Other operating systems like Windows will blue screen if you attempt to bring up
-a VirtualBox VM with Hyper-V enabled. Below are a couple of ways to ensure you
-can use Vagrant and VirtualBox if another hypervisor is present.
-
-### Linux, VirtualBox, and KVM
-
-The above error message is because another hypervisor (like KVM) is in use.
-We must blacklist these in order for VirtualBox to run correctly.
+You must denylist the additional hypervisors in order for VirtualBox to run correctly.
```

Comment:
```suggestion
You must add the additional hypervisors to the deny list in order for VirtualBox to run correctly.
```
`Denylist` as a verb sounds odd to me but feel free to keep it as I learned today that it is, in fact, a verb 😅